src/plugins/dashboard/config.ts

```diff
@@ -20,7 +20,7 @@
 import { schema, TypeOf } from '@kbn/config-schema';
 
 export const configSchema = schema.object({
-  allowByValueEmbeddables: schema.boolean({ defaultValue: false }),
+  allowByValueEmbeddables: schema.boolean({ defaultValue: true }),
```

Comment:
🎉

Reply:
Just curious, is the plan to enable this in 7.12 and then delete everything in 7.13? Or are you also planning for a cleanup in a separate PR?

Reply:
That's still up in the air - because removing the ability to turn the feature off _could_ constitute a breaking change. Right now the consensus seems to be removing references to it in 8.0.0

Reply:
We'll craft a plan to *at least* deprecate with a warning in 8.0 for removal in a follow-on minor, if not remove outright.  It kind of depends on feedback from the field.